fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/common/StoreTimer.java

```diff
@@ -376,22 +378,32 @@ public interface Wait extends StoreTimer.Event {
 
     /**
      * {@link Event}s that only count occurrences or total size.
-     * There is no meaningful time duration associated with these events.
+     * There is no meaningful cumulative value associated with these events.
      */
     public interface Count extends StoreTimer.Event {
         /**
          * Get whether the count value is actually a size in bytes.
          *
          * @return {@code true} if the count value is actually a size in bytes
          */
-        boolean isSize();
+        default boolean isSize() {
+            return false;
+        }
     }
 
     /**
-     * Contains the number of occurrences and cummulative time spent on an associated {@link StoreTimer.Event}.
+     * {@link Event}s that only count the number of occurrences of a read/write operation and also the size read/written.
+     * There is no meaningful time duration associated with these events.
+     */
+    public interface SizeEvent extends StoreTimer.Event {
```

Comment:
There should be some new tests in `FDBStoreTimerTest` that `SizeEvent`s do the right thing for:
1. `recordSize`
2. snapshot
3. diff
4. clear

Reply:
added